packages/functional-tests/pages/login.ts

```diff
@@ -52,6 +54,8 @@ export class LoginPage extends BaseLayout {
     LINK_USE_RECOVERY_CODE: '#use-recovery-code-link',
     NUMBER_INPUT: 'input[type=number]',
     PASSWORD: '#password',
+    SHOW_PASSWORD: '#password ~ [for="show-password"]',
+    SHOW_VPASSWORD: '#vpassword ~ [for="show-vpassword"]',
```

Comment:
These two lines are identical to the two above.  Why are there two selector maps?

Reply:
Good catch. Remove the duplicate lines.Thanks!